packages/jellyfish-api-core/__tests__/category/loan/paybackLoan.test.ts

```diff
@@ -315,7 +315,67 @@ describe('paybackLoan success', () => {
     expect(vaultAfter.informativeRatio).toStrictEqual(-1)
 
     const bobLoanAccAfter = await bob.rpc.account.getAccount(bobloanAddr)
-    expect(bobLoanAccAfter).toStrictEqual(['4.99995432@TSLA'])
+    expect(bobLoanAccAfter).toStrictEqual(['4.99995433@TSLA'])
+
+    // generate another 10 blocks and confirm that vaults does not accumulate interests once paid back fully
+    {
+      await bob.generate(10)
+      const vaultWayAfterPayback = await bob.container.call('getvault', [bobVaultId])
+      expect(vaultWayAfterPayback).toStrictEqual(vaultAfter)
+    }
+  })
+
+  it('should paybackLoan when loan amount is only 1 sat', async () => {
```

Comment:
check the burn address also. ideally, the burn address should not get anything added due to this test.

Reply:
Also worth adding 1 sat DFI payback test. It's touching extra paths in cpp payback code.